pkg/handler/common/provider/aws.go

```diff
@@ -257,6 +263,11 @@ func AWSSizes(region string, quota kubermaticv1.MachineDeploymentVMResourceQuota
 	return filterAWSByQuota(sizes, quota), nil
 }
 
+func isARM64Architecture(physicalProcessor string) bool {
+	// right now there is only one Arm-based processors: Graviton2
+	return strings.Contains(physicalProcessor, "Graviton")
```

Comment:
I'd stick to names here like I said in the https://github.com/kubermatic/kubermatic/pull/6927. If you want to keep it then please adjust the comment above.

Reply:
https://github.com/kubermatic/kubermatic/pull/6927#discussion_r620431596

Reply:
Ignore my comment.